sms-hello-world.js

```diff
@@ -8,12 +8,16 @@ const freeclimbSDK = require('@freeclimb/sdk')
 const port = process.env.PORT || 80
 const accountId = process.env.ACCOUNT_ID
 const apiKey = process.env.API_KEY
-const freeclimb = freeclimbSDK(accountId, apiKey)
+const freeclimbConfig = freeclimbSDK.createConfiguration({accountId, apiKey})
+const apiInstance = new freeclimbSDK.DefaultApi(freeclimbConfig);
 
 app.post('/incomingSms', (req, res) => {
-  let to = 'example to number'
-  let from = 'example from number'
-  freeclimb.api.messages.create(from, to, 'Hey! It is your application!').catch(err => {console.log(err)})
+    messageRequest = {
+      _from: '', // Your FreeClimb Number 
+      to: '', // Number to receive message
```

Comment:
could we set the 'to' number to the number that initiated the /incomingSms request (the 'from' field on the request body ex. https://gitlab.vailsys.com/kbrune/freeclimb-wordle/-/blob/master/routes/incomingSms.js#L9) - i think it'll help demonstrate that the /incomingSms webhook returns information to the application about who is texting